ibis/common/geospatial.py

```diff
@@ -35,14 +35,18 @@ def _format_geo_metadata(op, value: str, inline_metadata: bool = False) -> str:
     srid = op.args[1].srid
     geotype = op.args[1].geotype
 
-    if inline_metadata and srid:
-        value = "'SRID={};{}'".format(srid, value)
-    else:
-        value = "'{}'".format(value)
+    if inline_metadata:
+        value = "'{}{}'{}".format(
+            'SRID={};'.format(srid) if srid else '',
+            value,
+            '::{}'.format(geotype) if geotype else ''
+        )
+        return value
+    value = "'{}'".format(value)
```

Comment:
Use `repr(value)` here if `value` is a string instead of hand embedding quotes.

Reply:
sounds good! I will try that!